src/firefly/js/visualize/CsysConverter.js

```diff
@@ -295,8 +327,17 @@ export class CysConverter {
      * returns {ImagePt}
      */
     static makeIPtFromIWPt(iwPt) {
-        if (!iwPt) return null;
-        return makeImagePt(iwPt.x, iwPt.y);
+        return iwPt ? makeImagePt(iwPt.x, iwPt.y) : null;
+    }
+
+    static makeIPtFromFitsImPt(pt) {
+        return pt ? makeImagePt(pt.x-.5, pt.y-.5) : null;
+    }
+
+    makeIPtFromxZeroImPt(pt) {
+        if (!pt) return null;
+        const {ltv1,ltv2}= CysConverter.getLtv(this.projection.header);
+        return makeZeroBasedImagePt(pt.x+.5+ltv1, pt.y+.5+ltv2);
```

Comment:
does here make {type:ZERO_BASED_IM_PT} or {type:IM_PT}?  should that be the second one instead of the first one?

Reply:
good catch, thanks.